ui/src/components/NavigationBar/MainMenu.js

```diff
@@ -137,17 +137,16 @@ const MainMenu = ({ login, onClose }) => {
                 >
                   <Button marginBottom="5" width="100%">
                     <LinkOverlay href="/profile" onClick={onClose}>
-                      My Profile
+                      MY PROFILE
```

Comment:
all these buttons do not work when i press the actual button itself. have to click the text.